README.md

```diff
@@ -10,20 +10,32 @@ There are three main components in this repository: the circuit layout (file [ci
 
 For building the measurement device, you need the following parts:
 - Arduino Mega 2560: Does not have to be original Arduino, can also be e.g. a SunFounder Mega 2560
-- LED: A Light-emitting diode, e.g. LED 5-4500 RT
-- Phototransistor: For example the SDP 8406-003
-- Resistor 11kOhm: Any 11kOhm resistor will do the job, e.g. the 1/4W 11K
+- LED: A Light-emitting diode, e.g. `LED 5-4500 RT`. You might need to play around with these - depending on the colour and brightness of the LED, the PT may have a harder time detecting the LED on your screen
+- Phototransistor: For example the `SDP 8406-003`
+- Resistor 11k$\Omega$: Any 11k$\Omega$ resistor will do the job, e.g. the 1/4W 11K
+- Resistor 220$\Omega$: A resistor around this value will limit the current draw from the LED and protect the Arduino pin
 - Wires
-- Optional: A breadboard
+- Optional: A breadboard, protoboard or Arduino shield
```

Comment:
Can you state what you're using in the images? Arduino shield, I guess? Which one specifically?